libvast/src/format/pcap.cpp

```diff
@@ -27,7 +26,7 @@
 #  include <caf/settings.hpp>
 
 #  include <cstddef>
-#  include <pcap.h>
```

Comment:
@dominiklohmann looks like a rebase error in your top-commit. I'll rebase and push to this branch in a minute.

Reply:
Yes sorry, that shouldn't have been removed. Thanks for the quick response. We were just looking at which PRs to merge in what order now that they're unblocked.

We gave you a shoutout in the release notes and the release blog post btw. 😊

Reply:
No problem, this PR should be good now.

Thanks for the kind words in the release notes and blog post -- just gave them a read 😊